dev_environment.yml

```diff
@@ -14,6 +14,8 @@ dependencies:
   - pkg-config
   - make
   - ninja
+  # dev tools
+  - pre-commit
```

Comment:
added pre-commit to our default environment (others will still have to install manually unless recreating from scratch)